src/activation/softmax.rs

```diff
@@ -77,4 +77,8 @@ where
     fn forward(&self, input: &Tensor<D>) -> Tensor<D> {
         self.activate(input)
     }
+
+    fn name(&self) -> String {
+        String::from("softmax")
```

Comment:
Just to be consistent, should this be "Softmax" instead since other activation functions like ReLU are capitalized?

Reply:
Yep, missed that.

Reply:
It's a little strange because my local repo does have "Softmax" and I did add all changed files for the commit.